docs/strat-lib/getting-started/smart-offer.md

```diff
@@ -160,81 +175,44 @@ Gas used: 168639
 
 `0x0000000000000000000000000000000000000000000000000000000000000235` is the offer id.
 
-### Unlocked liquidity (%%reactive liquidity|reactive-liquidity%%)
-
-Notice that the offer was posted without transferring tokens from the admin to Mangrove or the `OfferMakerTutorial`. This way the tokens are pulled just-in-time when the offer is taken and can thus be made available for other purposes.
-
-For this to work, we need to let the `OfferMakerTutorial` pull funds from the admin's reserve of WETH.
-
-```bash
-cast send --rpc-url $LOCAL_URL "$WETH" "approve(address, uint)" "$OFFER_MAKER" 1000000000000000000 --private-key "$PRIVATE_KEY"
-```
-
-Alternatively, the admin could transfer tokens to the contract and lock them until the offer is taken or the tokens are withdrawn.
-
-The `OfferMakerTutorial` uses the approval to transfer funds from the admin, but this could also involve a %%router|router%% and require additional approvals depending on the scenario. See [approvals](../guides/approvals.md) for more details.
-
-#### Mint
```

Comment:
where did the mint stuff go? it is important to be able to execute offers.

Reply:
Thanks, I'm adding them back (tested it for my token pair, works fine 👍 )